toptal/picasso
# feat(input): new prop to enable manual resize for multiline

new prop **multilineResizable** that allows the user to manually resize a multiline input

[BT-450](https://toptal-core.atlassian.net/browse/BT-450)

### Description

As requested on [BT-450](https://toptal-core.atlassian.net/browse/BT-450), some multiline inputs needs to have the native browser's textarea handler on the bottom-right, so the user can manually change the size of the textarea.

Due to design restrictions the resize handler is only allowed to works **vertically**.

**Auto-expand support:** This new prop (**multilineResizable**) also works with **rows** and **rowsMax** props enabling auto-expand + manual resize on the same component. There is a specific example for that behaviour on the storybook.

### How to test

Opt 1:
- Just add **multilineResizable** to any multiline input
- A resize handler should appear on the bottom-right
- The user can resize the textarea using the resize handler
- Auto-expand should also work if enabled (using rows and rowsMax props)

Opt 2:
- Check the new "_Multiline expand and resize | Textarea_" section on the storybook.

**Regression:** Any input without the new prop should not be affected.

### Screenshots

| Before.                                    | After.                                       |
| --------------------------------------- | --------------------------------------- |
| https://clb.sh/a11915               | https://clb.sh/b7ddce              |

### Review
